libminifi/include/utils/GeneralUtils.h

```diff
@@ -61,6 +61,17 @@ T exchange(T& obj, U&& new_value) {
 template<typename...>
 using void_t = void;
 
+template<class T>
+using remove_reference_t = typename std::remove_reference<T>::type;
+
+template<class T>
+using remove_cv_t = typename std::remove_cv<T>::type;
+
+template<class T>
+struct remove_cvref {
+    typedef remove_cv_t<remove_reference_t<T>> type;
+};
```

Comment:
Please move `remove_cvref` either outside of `#ifdef` checks, or another one that checks for C++20 and falls back to `std::remove_cvref` if >= C++20.

Reply:
As requested.